### PR TITLE
rdgo: Use assembler container

### PR DESCRIPTION
**Pull request by cgwalters**
We need to do this because Python 3.  One step for making this
pipeline more like FAHC.

### CODE REVIEW

File: centos-ci/run-rdgo [hunk 1]
```diff
@@ -5,12 +5,13 @@ basedir=$(cd $(dirname $0) && pwd)
 . ${basedir}/libtask.sh
 
 cd rdgo
-ln -sf ${buildscriptsdir}/overlay.yml .
+ln -sfr ${buildscriptsdir}/overlay.yml .
```
Comment (jlebon) [inline centos-ci/run-rdgo]: Won't this symlink be broken inside the container?
Sounds like this should be `/srv/src/overlay.yml`?

Comment (cgwalters) [reply centos-ci/run-rdgo]: But why does it work for FAHC?   Anyways, yeah might as well just do that.

Comment (jlebon) [reply centos-ci/run-rdgo]: FAHC has

Comment (cgwalters) [reply centos-ci/run-rdgo]: Ugh, how did I miss that?  I must have been looking that the wrong copy of the file?  Anyways done :arrow_down: and it looks like it's working.

**Comment by jlebon** [comment]
LGTM!

**Comment by cgwalters** [comment]
I had to do https://pagure.io/fedora-atomic-host-continuous/c/4daa5a06143acda4764dd88e91c3f58116cdc381?branch=master which we probably need here too.

**Comment by jlebon** [comment]
OK, I opened #350, though it'd be nice if we could unify FAHC and CAHC some more.

**Pull request merged**